Kubernetes/windows/helper.v2.psm1

```diff
@@ -710,12 +698,11 @@ Update-CNIConfig
             "cniVersion": "0.2.0",
             "name": "<NetworkMode>",
             "type": "flannel",
+            "capabilities": {
+                "dns" : true
```

Comment:
can you also add portmapping capabilities ?

Reply:
Will do in followup

Reply:
Portmapping isn't implemented in the containernetworking/plugins binaries.